src/dbnode/storage/shard.go

```diff
@@ -860,6 +860,8 @@ func (s *dbShard) purgeExpiredSeries(expiredEntries []*lookup.Entry) {
 		// The contract requires all entries to have count >= 1.
 		if count < 1 {
 			s.logger.Error("purgeExpiredSeries encountered invalid series read/write count",
+				zap.String("namespace", s.namespace.ID().String()),
```

Comment:
nit: probably using `zap.Stringer` would be better instead off calling `String()` methods directly. Same applies for `series` param.

Reply:
Good point, switched to `zap.Stringer`. Can you approve it?